src/test/java/org/apache/sysds/test/functions/builtin/part1/BuiltinImageMirrorLinearizedTest.java

```diff
@@ -0,0 +1,81 @@
+
+package org.apache.sysds.test.functions.builtin.part1;
+
+import org.junit.Test;
+import org.apache.sysds.common.Types.ExecMode;
+import org.apache.sysds.common.Types.ExecType;
+import org.apache.sysds.runtime.matrix.data.MatrixValue;
+import org.apache.sysds.test.AutomatedTestBase;
+import org.apache.sysds.test.TestConfiguration;
+import org.apache.sysds.test.TestUtils;
+
+import java.util.HashMap;
+
+public class BuiltinImageMirrorLinearizedTest extends AutomatedTestBase {
+    private final static String TEST_NAME_LINEARIZED = "image_mirror_linearized";
+    private final static String TEST_NAME = "image_mirror";
+    private final static String TEST_DIR = "functions/builtin/";
+    private final static String TEST_CLASS_DIR = TEST_DIR + BuiltinImageMirrorLinearizedTest.class.getSimpleName() + "/";
+    
+   private final static double eps = 1e-10;
+   private final static int rows = 64; 
+   private final static int cols = 64; 
```

Comment:
This is very small, and same number of rows and cols, please add tests cases for uneven images.

Reply:
my initial thought was we could test it for infinite amount of variables by  easily changing the values of the variables instead of having couple of concrete test cases. 64 here was just a place holder